fixtures/workers-with-assets/tests/index.test.ts

```diff
@@ -29,22 +29,26 @@ describe("[Workers + Assets] `wrangler dev`", () => {
 		expect(text).toContain(`<p>Learn more about Workers with Assets soon!</p>`);
 	});
 
-	it("should not resolve '/' to '/index.html' ", async ({ expect }) => {
-		let response = await fetch(`http://${ip}:${port}/`);
-		expect(response.status).toBe(404);
-	});
-
-	it("should 404 if asset is not found in the asset manifest", async ({
+	it("should fallback to the user worker if there are no assets at a given path ", async ({
 		expect,
 	}) => {
-		let response = await fetch(`http://${ip}:${port}/hello.html`);
-		expect(response.status).toBe(404);
-
-		response = await fetch(`http://${ip}:${port}/hello.txt`);
-		expect(response.status).toBe(404);
+		// Requests should hit the asset worker *first*, then try the user worker
```

Comment:
`Asset Worker`

Reply:
basically what I'm trying to say is that worker should be `Worker` when referring to CF Workers